preview-env/clean/graphql/query_get_label_id.gql

```diff
@@ -0,0 +1,9 @@
+query get_label_id ($owner:String!,$repo:String!,$label:String!) {
+
+  repository (name: $repo, owner: $owner) {
+    label (name: $label) {
+      id
+    }
+  }
+  
+}
```

Comment:
newline missing, just for completeness.
Would recommend adding pre-commit locally `pre-commit install` to run on this repo.

Reply:
Good point.